.travis.yml

```diff
@@ -20,50 +8,96 @@ env:
-    script: yarn run deploy:$COZY_APP_SLUG
-    on:
-      tags: true
-sudo: required
+    - secure: lggtB/s2BGWm0f/6aqWq9KoH4PgFdrWv5fR2QUKrWjeUjPus8uTG4WDfxgRRPF4GgUIkYcboODB43A01UzgbRi2KWZjysj0W8/oKj2bt1VqMtA+G6qQMXIIZ0d1W5+x38MsEmUnDamCMQEv38xrFK9xlStIfA/dCvJg+dYBhaGDxYcdSkA3IWVHRfrFXVLol/O+V+CvpSv5vP8AkTHgiILRUEb0nJxx2Dpoeecb79Qg/lAAhUOGz51VPbhms+t+W9DmAH+El9wrbYkxCTe/tyCQXzzXwV4jspz9/QO7nODaHOERrW8ZB6zFe8R+7nvDOtflunaPaa3trt7n93bbthoDIDgOPjD1Gud2Hh7Vsp6P0yYCC3nAj1El5ueK0FIeE7TbltVZFxNqh67ULruZWC1zt1kA3WmBohZthw6f0XGiOnLD1J7O4rn9SPR6t6WFO3LSkl8/jJ250NdcWG+jl7qsF5Brk+9SSZRdFodHPx6EzggqzlKCVfduOYJnYAVf8vAeEvPaXvSdX34iHLzeG80vaanDOCWWlviLGmxojf4a9MteAJTRKwfcozQYF8yf13gtUlRPDw7eq7Fj4oRCoV0aDF5qy0V8w1zpTtPZXJ4BpNMWMMR9Hs+UlVBFwAGw+01Xdk4S1cSEgVbkl5idlUSbiDYQtiZARPQW+Z3Z/s5s= #ANDROID_KEYSTORE_PASS
+node-preset: &node # Used internally to share configuration, ignored by travis.
+  language: node_js
+  node_js:
+    - '8'
+  cache:
+    yarn: true
+    directories:
+      - node_modules
+stages:
+  - prebuild
+  - build
+jobs:
+  include:
+    - name: "Lint"
+      stage: "prebuild"
+      <<: *node
+      script: yarn lint
+    - name: "Unit tests"
+      stage: "prebuild"
+      <<: *node
+      script: yarn test
+    - &build-web
+      name: "Drive web"
+      stage: "build"
+      <<: *node
+      env:
+        - COZY_APP_SLUG=drive
+      addons:
+        firefox: latest
+      before_install:
+        - scripts/transifex.sh
+        - scripts/decrypt.sh
+      script:
+        - yarn tx
+        - yarn build:$COZY_APP_SLUG:browser
+        - test TRAVIS_PULL_REQUEST && yarn testcafe:$COZY_APP_SLUG
+      before_deploy:
+        - &downcloud-cert if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then eval "$(ssh-agent -s)" && chmod 600 /tmp/id_rsa_travis_downcloud && ssh-add /tmp/id_rsa_travis_downcloud; fi # Needs to be inlined and not in a script to work.
+        - yarn add cozy-app-publish
+      deploy:
+        - &deploy
+          provider: script
+          repo: cozy/cozy-drive
+          skip-cleanup: true
+          script: yarn run deploy:$COZY_APP_SLUG
+          on:
+            branch: master
+        - <<: *deploy
+          on:
+            tags: true
+    - <<: *build-web
+      name: "Photos web"
+      env:
+        - COZY_APP_SLUG=photos
+    - name: "Drive android"
+      stage: "build"
+      language: android
+      android:
+        components:
+          - build-tools-26.0.2
+      before_cache:
+        - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+        - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+      cache:
+        yarn: true
+        directories:
+          - node_modules
+          - $HOME/.gradle/caches/
+          - $HOME/.gradle/wrapper/
+          - $HOME/.android/build-cache
+      env:
+        - COZY_APP_SLUG=drive
+      before_install:
+        - yes | sdkmanager "platforms;android-27" # see https://github.com/travis-ci/travis-ci/issues/8874#issuecomment-350350607
```

Comment:
You use android-27 here, but build with build-tools-26 ? https://github.com/cozy/cozy-drive/pull/1418/files#diff-354f30a63fb0907d4ad57269548329e3R71